cfi_intro.adoc

```diff
@@ -43,247 +44,163 @@ address of the instruction following the jump (`pc+2`) to the link register `x1`
 and is a procedure call. The `C.JR` is a return from procedure if `rs1` is a
 conventional link register (i.e. `x1` or `x5`); else it is an indirect jump.
 
-The RISC-V control-flow integrity (CFI) extension (Zicfisslp) builds on these
-conventions and hints.
-
-The term "call" is used to refer to a `JAL` or `JALR` instruction with a link
+The term _call_ is used to refer to a `JAL` or `JALR` instruction with a link
 register as destination, i.e., `rd != x0`. Conventionally, the link register is
-`x1` or `x5`. A call using `JAL` or `C.JAL` is termed a direct call. A `C.JALR`
-expands to `JALR x1, 0(rs1)` and is a call. A call using `JALR` or `C.JALR` is
-termed an indirect call.
+`x1` or `x5`. A _call_ using `JAL` or `C.JAL` is termed a direct call. A
+`C.JALR` expands to `JALR x1, 0(rs1)` and is a _call_. A _call_ using `JALR` or
+`C.JALR` is termed an _indirect-call_.
 
-The term "return" is used to refer to a `JALR` instruction with `rs1 == x1` or
+The term _return_ is used to refer to a `JALR` instruction with `rs1 == x1` or
 `rs1 == x5` and `rd == x0`. A `C.JR` instruction expands to `JALR x0, 0(rs1)`
```

Comment:
reworded to avoid ambiguity. 
This section was a recap of the architecture. The Zicfilp treatment of semantically direct call/jump is discussed later.